operator/operator.go

```diff
@@ -67,15 +67,23 @@ func New(
 
 // Setup creates an operator from the given configuration.
 func Setup(cfg *Config) *Operator {
```

Comment:
I'd rename this to `SetupWithPrototypeEnclave` and `SetupWithEnclave` to `Setup`.